helm/efk-stack-app/.abs/main.yaml

```diff
@@ -0,0 +1,8 @@
+replace-chart-version-with-git: true
```

Comment:
It this file correctly placed here? I thought it needs to be placed at the root of a repository? (`.abs/main.yaml`)

I'm running `dabs.sh` with container tag `0.1.2-7a6961a0338b59f11392a89a04baa621c226682d` in the root of the repository with flags `--steps all --chart-dir helm/efk-stack-app/` and it seems the configuration from there is not picked up..

Moving it to `.abs/main.yaml` works though.